src/containers/Secrets/Secrets.js

```diff
@@ -237,8 +237,10 @@ export /* istanbul ignore next */ class Secrets extends Component {
       });
     } else {
       secrets.forEach(secret => {
-        if (secret.data.accessToken) {
-          secretsToUse.push(secret);
+        if (secret.data) {
```

Comment:
can we combine these and avoid the nesting? 🙏 

`if (secret.data && secret.data.accessToken) {`

Reply:
On it!

Reply:
Done - and the Prow PR just went in too so...happy days 🤞